### ISSUE TITLE
Re: Shutdown

### ISSUE THREAD

**Issue by exikyut** [mikecrittenden/toogles#31]
Just learned about the API situation while scrolling your website's post history (after reading another linked article from HN). So annoying when that happens...

You may already know about https://invidio.us/, which provides a YouTube frontend by going in the opposite direction to what this project does, and HTML-scraping youtube.com.

I'm not sure exactly how one server (??) hits YouTube on behalf of apparently 35k users without being nuked, but... somehow... it lives.

Incidentally, while verifying I did indeed remember the website's name, I discovered that the author has hit burnout: https://omar.yt/posts/stepping-away-from-open-source

It seems the server-scraping approach scales (further clarification would probably be needed, and the details probably quite interesting) where the API approach, er, doesn't.

I have no particular investment in either project; I'm simply mentioning invidio.us in case you find it relevant/interesting.

**Comment by cloudmemory** [comment]
Has toogl.es been shut down permanently?

**Comment by mePy2** [comment]
Any news?

**Comment by mikecrittenden** [comment]
Yes, it's shut down permanently unfortunately. Here's some more info: https://critter.blog/2020/08/04/a-farewell-to-toogles/

**Comment by fadster** [comment]
google is slowly but surely destroying the Internet. :frowning_face:
